app/models/response.rb

```
@@ -50,25 +81,87 @@ def reportable_difference?
     sum = 0
+    # E2619: quiz maps are identified by reviewer_id == reviewee_id (the student reviews
+    # themselves). The response_maps table has no STI type column so is_a?(QuizResponseMap)
+    # always returns false; this is the only reliable discriminator.
+    is_quiz = map.reviewer_id == map.reviewee_id
+    # E2619: The frontend stores question types with spaces ("Text field", "Multiple choice",
+    # "Multiple choice checkbox"). Include both the spaced and CamelCase variants so scoring
+    # works regardless of which convention was used when the quiz was created.
+    comment_scored_types = %w[
+      TextField MultipleChoiceRadio MultipleChoiceCheckbox
+      Text\ field Multiple\ choice Multiple\ choice\ checkbox
+    ].freeze
```

RuboCop (1.86.1)

[convention] Use an array literal `[...]` for an array of words.

(Style/WordArray)